ci/azure-pipelines.yml

```diff
@@ -43,11 +44,11 @@ jobs:
 - job: linux_python37
   pool:
     vmImage: ubuntu-18.04
-  container: python:3.7-buster
+  container: quay.io/pypa/manylinux1_x86_64
   variables:
     interpreter: python3.7
     manylinux: 1
-    path: /home/vsts_azpcontainer/.local/bin:/home/vsts_azpcontainer/.cargo/bin:/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+    path: $HOME/.local/bin:$HOME/.cargo/bin:/opt/python/cp37-cp37m/bin:/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment:
I tried using `$HOME` originally but it didn't expand when used as a variable. Do you not need `.local/bin` for the `--user` install?

Reply:
What do you mean with "didn't expand"? Azure Pipelines creates a bash script from the code. Is $HOME not set? I was unable to find the `/home/vsts_azpcontainer` value in the documentation (quick search). If $HOME is not set, maybe `~` will do the trick.

Yeah, `.local/bin` is required only due to `--user`. Dependencies could be installed globally.